chainer/functions/noise/simplified_dropconnect.py

```diff
@@ -25,9 +25,10 @@ class SimplifiedDropconnect(function.Function):
 
     """Linear unit regularized by simplified dropconnect."""
 
-    def __init__(self, ratio, mask=None):
+    def __init__(self, ratio, batchwise_mask, mask=None):
```

Comment:
I am afraid it breaks compatibility because if a user code that feeds `mask` as a positional argument accept that value as `batchwise_mask` in this method.